frontend/src/views/ProfileView.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ProfileView from './ProfileView';
+
+describe('Profile View', () => {
+    var wrapper;
+    beforeEach(() => {
+        wrapper = shallow();
+    });
+
+    describe('#handleSubmit()', () => {
+        it('', () => {
+        });
+    });
+});
```

Comment:
These tests seem to have been prepared, but aren't actually written.  Consider expanding this a bit to cover the functionality we'd expect from the profileView.js file.